dbt-tests-adapter/dbt/tests/adapter/utils/test_cast.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+
+from dbt.tests.adapter.utils import base_utils, fixture_cast
+
+
+class BaseCast(base_utils.BaseUtils):
+    @pytest.fixture(scope="class")
+    def seeds(self):
+        return {"data_cast.csv": fixture_cast.seeds__data_cast_csv}
+
+    @pytest.fixture(scope="class")
+    def models(self):
+        return {
+            "test_cast.yml": fixture_cast.models__test_cast_yml,
+            "test_cast.sql": self.interpolate_macro_namespace(
+                self.interpolate_macro_namespace(
+                    fixture_cast.models__test_cast_sql, "cast"
+                ),
+                "type_string",
+            ),
+        }
+
+
+class TestCast(BaseCast):
```

Comment:
I don't think we need this class, do we? This would be created in the calling test suite. If you import it, there's no way to tell `pytest` not to run it. We used to add it when `dbt-postgres` was in `dbt-core` because we could just run this whole suite to run the tests for `dbt-postgres`, but that's no longer the case.

Reply:
My preference is for consistency.

And then if we want to take some kind of action, then this would be included along with that.

But I'd be happy to defer to your judgement if you prefer I peel it out.

Reply:
I just saw that you approved and gave me the option here:

> Your choice as to removing the test implementation or keeping it.

Since CI is passing, I'm going to merge now. But just let me know if you want me to peel it out in a follow-up PR!